packages/p2p/src/components/app.jsx

```diff
@@ -136,7 +136,7 @@ const App = observer(props => {
                         />
                     </div>
                 ) : (
-                    <Modal is_open={general_store.show_popup} className='p2p-nickname__dialog'>
+                    <Modal is_open={general_store.show_popup} width={440} className='p2p-nickname__dialog'>
```

Comment:
Is this okay for mobile too?

Reply:
yes, because we don't wrap <NicknameForm> component in <Modal> when it's mobile